types/auth0/index.d.ts

```diff
@@ -541,6 +541,23 @@ export interface CreateConnection extends UpdateConnection {
     strategy: Strategy;
 }
 
+export interface GetConnectionsOptions extends PagingOptions {
+    /** List of fields to include or exclude */
+    fields?: string[] | undefined;
```

Comment:
```suggestion
    fields?: string | string[] | undefined;
```
the test uses string literal (string value), not a list of strings

Reply:
I don't understand what test you are referring to…? But yes, you are right – both `string` and `string[]` are accepted here. I've updated the type.

Reply:
> I don't understand what test you are referring to…? But yes, you are right – both `string` and `string[]` are accepted here. I've updated the type.

one from the posted link:
https://github.com/auth0/node-auth0/blob/master/test/management/connections.tests.js#L127